src/development/coding-licences.md

```diff
@@ -20,6 +20,47 @@ This should be called `LICENCE` or `LICENCE.md`. “License” is the U.S. Engli
 
 Make sure the licence content is included in full, including the title “APACHE LICENSE, VERSION 2.0”, so that readers are quickly able to see which licence is being used.
 
+### Forked projects
+
+If you have forked a project that was licensed under Apache 2 the following additional changes are required:
+
+1. Any significant changes have been documented. This should suffice if the original commit history has been retained but you can also include attributions in the NOTICE file
+
+```text
```

Comment:
Can you put this in a reveal
https://nhsbsa.github.io/nhsbsa-digital-playbook/how-to-contribute/markdown/#expanding-details